src/openrct2/rct1/RCT1.h

```diff
@@ -466,6 +466,10 @@ struct rct1_peep : RCT12SpriteBase
     uint8_t favourite_ride_rating; // 0xFA
     uint8_t pad_FB;
     uint32_t item_standard_flags; // 0xFC
+    uint64_t GetItemFlags() const
+    {
+        return item_standard_flags | (static_cast<uint64_t>(item_extra_flags) << 32);
```

Comment:
```suggestion
        return item_standard_flags;
```

or just set directly from `item_standard_flags`.

Reply:
cool sounds good. Strange that CS would have space for the flags and then not use them. Must have been something that he decided wasn't required in the end.